bin/workloads/ml/linear/spark/run.sh

```diff
@@ -26,7 +26,7 @@ rmr_hdfs $OUTPUT_HDFS || true
 
 SIZE=`dir_size $INPUT_HDFS`
 START_TIME=`timestamp`
-run_spark_job com.intel.hibench.sparkbench.ml.LinearRegression --numIterations $NUM_ITERATIONS_LINEAR --stepSize $STEPSIZE_LINEAR $INPUT_HDFS
+run_spark_job com.intel.hibench.sparkbench.ml.LinearRegressionWithElasticNet ${INPUT_HDFS}
```

Comment:
If NUM_ITERATIONS_LINEAR & stepSize are no longer parameters, we should also modify `linear.conf`

Reply:
@gczsjdy  Thanks , I will modify it .

Reply:
If we intend to modify this with a future PR by @xwu99 ,

@minmingzhu Could you add a TODO here?